x-pack/plugins/infra/public/components/asset_details/tabs/profiling/profiling.tsx

```diff
@@ -38,6 +42,30 @@ export function Profiling() {
           <Flamegraph />
         </>
       ),
+      append: (
+        <Popover iconSize="s" iconColor="subdued" icon="questionInCircle">
+          <EuiText size="xs">
+            <FormattedMessage
+              id="xpack.infra.profiling.flamegraphInfoPopoverBody"
+              defaultMessage="The flamegraph is a visual representation of the functions that are consuming the most resources. Each function is represented by a rectangle, where the width of the rectangle represents the amount of time spent in the function, and the number of stacked rectangles represents the stack depth. The stack depth is the number of functions that were called to reach the current function. {learnMoreLink}"
```

Comment:
```suggestion
              defaultMessage="See a visual representation of the functions that consume the most resources. Each rectangle represents a function. The rectangle width represents the time spent in the function, and the number of stacked rectangles represents the number of functions called to reach the current function. {learnMoreLink}"
```

Reply:
LGTM @mdbirnstiehl 

@iogbole - you OK with it?